### ISSUE TITLE
Feature request: Hide the sender's email address on the download page

### ISSUE THREAD

**Issue by makinog3** [filesender/filesender#1461]
The email address of the user uploading the file may not be shown to the user downloading the file.

For example, when the file was uploaded by a personal account, but the notification is made on behalf of an organization rather than the personal. In this case, the "From" of the email is set to the organization's ML address when the announcement is made, but it is not good that the download page shows the personal email address instead of that ML address.
Note that we use Shibboleth authentication, so we always use FileSender with the personal account.

**Comment by monkeyiq** [comment]
I am thinking that we could use the `transfer_options` like other options.  One advantage to doing this is that the config.php can then set a default and set a forced option such is the below (it is true but the user can not change it).

```
$config['transfer_options'] = array(
...
        'hide_sender_email' => array(
            'available' => false,
            'advanced' => true,
            'default' => true
        ),
...
```

I have quite a small patch to do this update which allows default and forced settings. It seems I can push it to the PR but I have pushed to my fork so as not to push unannounced. https://github.com/filesender/filesender/commit/2c646e5e0a75d3b6af1b08f5b5ec374dd27f1663

I see that having the option over near the email address might be useful as a hint to the user that they might be leaking this information. My update moves it back into the same options panel. I am open to moving it back to near the email address. Well, open enough that this subsequent update does just that https://github.com/filesender/filesender/commit/100aebf73989641e9c6ce0a00d86a1fd6f4104d6

We would also want to update `docs/v2.0/admin/configuration/index.md` with some information including the above example to force this option on.

If this looks ok I am happy to try to push those updates to this PR or merge them after merging this as an update.

**Comment by monkeyiq** [comment]
I pushed the update to the PR. Always feels a bit odd pushing a change to a fork that the original PR was made from.

**Comment by makinog3** [comment]
Thank you for your review!
I’m taking vacation now because it’s national holidays in Japan(and Vietnam :-). So I’ll confirm it next week.

**Comment by monkeyiq** [comment]
Cool. Enjoy your holiday!

**Comment by makinog3** [comment]
I confirmed it and updated a few more codes along with your review, and added the document. https://github.com/filesender/filesender/pull/1462#commits-pushed-7573e35

**Comment by monkeyiq** [comment]
Merged! Thank you for the new option :)

**Comment by makinog3** [comment]
Thank you!

**Comment by monkeyiq** [comment]
This has been merged and will be in release 2.41. Thank you for your contributions to FileSender!
